plugin-attribution/src/main/java/com/mapbox/maps/plugin/attribution/AttributionDialogManagerImpl.kt

```diff
@@ -56,17 +60,29 @@ class AttributionDialogManagerImpl(
    *
    * @param attributionTitles an array of attribution titles
    */
-  private fun showAttributionDialog(attributionTitles: Array<String>) {
+  private fun showAttributionDialog(attributions: List<Attribution>) {
     val builder = AlertDialog.Builder(context)
     builder.setTitle(R.string.mapbox_attributionsDialogTitle)
-    builder.setAdapter(
-      ArrayAdapter(
-        context,
-        R.layout.mapbox_attribution_list_item,
-        attributionTitles
-      ),
-      this
-    )
+    val adapter: ArrayAdapter<Attribution> = object : ArrayAdapter<Attribution>(
+      context,
+      R.layout.mapbox_attribution_list_item, attributions
+    ) {
+      override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+        val view = super.getView(position, convertView, parent)
+        val attribution = attributions[position]
+        view.findViewById<TextView>(android.R.id.text1).apply {
+          // if attribution url is empty, we show them as disabled.
+          if (attribution.url.isEmpty()) {
+            setTextColor(Color.GRAY)
+          } else {
+            setTextColor(ContextCompat.getColor(context, R.color.mapbox_blue))
```

Comment:
is it necessary to set the blue color again? I think the default style defined in `mapbox_attribution_list_item` is `mapbox_blue` already.

Reply:
yes, somehow the color `GRAY` was set to whole list, so added the condition.